src/components/header/Header.tsx

```diff
@@ -18,11 +31,13 @@ export class Header extends React.Component<HeaderProps> {
             </div>
             <div className="right-container">
                 {!this.props.isLoginPage && (
-                    <button className="sign-out-button" aria-label={"Sign out"}>
+                    <button className="sign-out-button" onClick={this.onSignOutClick} aria-label={"Sign out"}>

```

Comment:
You can pass the function directly in here
```suggestion
                    <button className="sign-out-button" onClick={this.props.setLoggedOut} aria-label={"Sign out"}>

```